docs/gambit/mutation-verifier.md

```diff
@@ -35,129 +34,205 @@ If you are on Linux, you will need pip version 20.3 or above.
 
 ## Running the Mutation Verifier
 
-Once you have updated your `certora-cli` installation using `pip` to get the relevant
-dependencies, run Gambit from the command line:
+The mutation verification script is called `certoraMutate`.
+Run it from the command line:
 
-```
-certoraMutate --prover_conf path/to/prover/prover.conf --mutation_conf path/to/mutation/mutation.conf
+```sh
+certoraMutate --prover_conf path/to/prover.conf --mutation_conf path/to/mutation.mconf
 ```
 
+If you do, the script will generate code mutants, 
+  then submit a verification job per mutant to Certora's server.
+
 ```{note}
 You must run `certoraMutate` from the root of the Solidity project directory.
-The files `prover.conf` and `mutation.conf`
-can be in their own directories, but must always be within the project directory.
+The files `prover.conf` and `mutation.conf` can be in their own directories, 
+  but must always be within the project directory.
 All paths in `mutation.conf` are relative to the parent directory containing `mutation.conf`.
-Paths in `prover.conf` are all relative to the project directory's root, which is assumed to be the working directory.
+Paths in `prover.conf` are all relative to the project directory's root, 
+  which is assumed to be the working directory.
 ```
 
-## Configurations
-The tool expects two separate configuration files:
-the configuration file which defines the execution of mutant generation (`--mutation_conf`),
-and the configuration file which defines execution of the Prover (`--prover_conf`).
-Here is a simple configuration file setup using the example above:
-
-In `prover.conf`:
+When it finishes successfully, you should see the following lines printed:
 
-```json
-{
-  "files": [
-    "C.sol"
-  ],
-  "verify": "C:c.spec"
-}
 ```
-In `mutation.conf`:
+********************  PROVER END  ********************
+You will receive an email notification when this mutation test is completed. It may take several hours.
+```
+
+Verifying all the mutants is a heavy computational process that may take several hours.
```

Comment:
can we make it less scary? something like

"The time for verifying all the mutants depends on many factors like: the number of mutants,  the complexity of the contracts, the complexity of the spec, usage of filters etc. In extreme  cases the process from start to finish may even take several hours

Reply:
good point, added